README.md

```diff
@@ -520,6 +527,7 @@ Services to securely store your Docker images.
 
 ## Newsletter
 * [Docker Team](https://www.docker.com/)
+* [Codefresh](https://codefresh.io/blog/)
```

Comment:
See my [comment above](https://github.com/veggiemonk/awesome-docker/pull/396/files#r129782704). We also may remove this category, as we would like to avoid redundancy within the `Useful Resources` section.